ci/tasks/check-if-usns-are-applicable.sh

```
@@ -0,0 +1,42 @@
+if [[ -n "${DEBUG:-}" ]]; then
+  set -x
+  export BOSH_LOG_LEVEL=debug
+  export BOSH_LOG_PATH="${BOSH_LOG_PATH:-${REPO_PARENT}/bosh-debug.log}"
+fi
+
+source "${REPO_ROOT}/ci/tasks/usn-processing/usn-shared-functions.sh"
+
+packages_included_in_stemcell=false
+mapfile -t FOUND_USNS < "${REPO_PARENT}/usns/usns.json"
+for usn in "${FOUND_USNS[@]}"; do
+  id=$(echo $usn | jq -r '.url' | cut -d '/' -f6 | cut -d '-' -f2,3)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)